testproject/Assets/Scenes/ZooSam.unity.meta

```diff
@@ -0,0 +1,7 @@
+fileFormatVersion: 2
```

Comment:
Maybe renamed to something more descriptive like `NetworkTransformInterpolationSample`

Reply:
yeah, "Zoo" was something we used in past projects where we had our own little sandboxes, with a controlled environment. They are not meant to be used by other devs, except for sharing progress and experiments. Maybe I shouldn't have committed it, but I didn't feel it would be harmful. I can remove it if we want

Reply:
bump

Reply:
I think we should keep it. I'm fine with the way it is but I'd still suggest to give it a more descriptive name instead of your own name so that we understand what this scene is for in the future. But I don't care much just a suggestion.